ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -0,0 +1,241 @@
+package org.cru.godtools.articles.aem.service.support;
+
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.model.Attachment;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+
+import timber.log.Timber;
+
+/**
+ * This class handles parsing any AEM json calls into DOA objects.
+ */
+public class ArticleParser {
+
+    public static final String ARTICLE_LIST_KEY = "article_list_key";
+
+    public static final String ATTACHMENT_LIST_KEY = "attachment_list_key";
+
+    private JSONObject articleJSON;
+
+    private List<Attachment> attachmentList = new ArrayList<>();
+
+    private List<Article> articleList = new ArrayList<>();
+
+    private String createdTag = "jcr:created";
+    private String contentTag = "jcr:content";
+    private String lastModifiedTag = "cq:lastModified";
+    private String uuidTag = "jcr:uuid";
+    private String titleTag = "jcr:title";
+    private String rootTag = "root";
+    private String fileTag = "fileReference";
+    private String baseUrl = "https://stage.cru.org";
```

Comment:
this shouldn't be here, where are you using it?

Reply:
you should be relying on the AEM import url for whatever functionality this is providing

Reply:
I am just using them to parse the json data.  I wanted to make them member variable so I can easily change them if needed.